charts/spire/charts/spire-server/templates/_controller-manager-container.tpl

```diff
@@ -75,7 +75,11 @@
     {{- end }}
   env:
     - name: ENABLE_WEBHOOKS
+    {{- if eq .Values.controllerManager.staticManifestMode "off" }}
       value: {{ .webhooksEnabled | toString | quote }}
+    {{- else }}
+      value: "false"
```

Comment:
The static manifest mode doesnt need webhooks?

Reply:
no. It reads the static manifests right from the file system. In this case, provided by a configmap.

Reply:
It can only do one at a time?

Reply:
Correct. it either pulls the yaml objects from a local directory (static manifests) or from k8s